# ARTIC-HELP: Extended Data Repository

This repository contains the **underlying data, extended data, and supplementary materials** for the manuscript:

**_‚ÄúReducing Supply Chain Dependencies for Viral Genomic Surveillance: Get by with a Little HELP from Commercial Enzymes Already in Your Lab Freezer‚Äù_**

> ‚úèÔ∏è _Kovalenko, G., Hosmillo, M., Kent, C., Rowe, K., Rambaut, A., Loman, N. J., Quick, J., & Goodfellow, I. (2025). Reducing supply chain dependencies for viral genomic surveillance: Get by with a little HELP from commercial enzymes already in your lab freezer. bioRxiv. https://doi.org/10.1101/2025.06.11.658579_

---

##  Contents
This repository includes the following files:

## Underlying Data
All consensus sequences generated in this study are available in the `ARTIC-HELP_consensus-fasta.zip` archive. This file contains the following directories:

- **`stage1_sars_cov_2_consensus/`**  
  SARS-CoV-2 consensus sequences from the stage 1, workflows 1‚Äì6.  
  Includes a control panel with four RNA concentrations derived from a lab-grown isolate  
  (*SARS-CoV-2/human/Liverpool/REMRQ001/2020*).

- **`stage2_sars_cov_2_consensus/`**  
  SARS-CoV-2 consensus sequences from the stage 2. Includes a control panel with four RNA concentrations  
  derived from a lab-grown isolate (*SARS-CoV-2/human/Liverpool/REMRQ001/2020*).

- **`stage3_sars_cov_2_consensus/`**  
  SARS-CoV-2 consensus sequences from the stage 3, clinical panel of 19 samples  
  processed using three workflows: **LoCost**, **HELP-Ie (Platinum)**, and **HELP-IIe (EcoDry)**.

- **`stage3_norovirus_gii_consensus/`**  
  Norovirus GII consensus sequences from the stage 3, clinical panel of 12 samples  
  processed using three workflows: **LoCost**, **HELP-Ie (Platinum)**, and **HELP-IIIe (Q5)**.

## Extended Data:

###  `ARTIC-HELP_Extended-data.xlsx`
This Excel file includes:

- **Table S1** ‚Äì HELP Master Mix Recipes.  
- **Table S2** ‚Äì Reagent List used in the HELP study.  
- **Table S3** ‚Äì Characteristics of polymerases used in this study (as provided by the supplier).  
- **Table S4** ‚Äì Sequencing quality control metrics (Stages 1 and 2: control panel: lab-grown isolate SARS-CoV-2/human/Liverpool/REMRQ001/2020).  
- **Table S5** ‚Äì Sequencing quality control metrics (Stage 3: 19 clinical SARS-CoV-2 samples).  
- **Table S6** ‚Äì Sequencing quality control metrics (Stage 3: 10 clinical Norovirus Genogroup II samples).  
- **Table S7** ‚Äì Cost comparison per sample for libraries containing 24, 48, and 96 samples.

###  `ARTIC-HELP_Figure_S1.png`
- **Figure S1** ‚Äì Stage 1: Gel electrophoresis results for M-MLV (wf-2) and six polymerases (wf-3) compared to ARTIC LoCost (wf-1).

---

##  Description

This data supports the evaluation of **ARTIC-HELP workflows**  
(**H**omebrew **E**nzymes for **L**ibrary **P**reparation), designed for amplicon-based sequencing using **commercial enzymes and buffers** as alternatives to standard reagents in the ARTIC-LoCost protocol and the nanopore native barcoding workflow (ONT), including RT, PCR, end-prep, barcode ligation, and adapter ligation.

Study Aim: Reducing supply chain dependencies for viral sequencing using ARTIC-HELP workflows

---

## üîó Citation

Please cite this dataset alongside the associated manuscript:

Kovalenko, G., et al. (2025). Reducing supply chain dependencies for viral genomic surveillance: Get by with a little HELP from commercial enzymes already in your lab freezer. bioRxiv. https://doi.org/10.1101/2025.06.11.658579 

---

## License
This work is licensed under a [Creative Commons Attribution 4.0 International License (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/).


---

##  Contact

anna31kovalenko@gmail.com
